### PR TITLE
fix crash when creating initials for user without first and last name

### PR DESCRIPTION
**Pull request by jivey**
https://openstax.atlassian.net/browse/CORE-587

Assignable students do not have first and last names associated, so REX crashes trying to make initials for the dropdown menu. Suggestions for alternatives to "AU" (Anonymous User? Assignable User?) are welcome.

### CODE REVIEW

**Comment by bethshook** [comment]
that's a tough one. my only other ideas are `ME` and `OS`

**Review by bethshook — APPROVED** [review]

**Comment by RoyEJohnson** [comment]
A question mark would be unambiguous.

**Review by RoyEJohnson — APPROVED** [review]

**Comment by TomWoodward** [comment]
if you made them spaces it would show an empty circle, i feel like even that would be less confusing than `AU`, i'm not sure what people are gonna think when they see that. only other thing i can think of is change the thing entirely to show like a profile icon (maybe only if there are no initials?)

File: src/app/components/NavBar/__snapshots__/index.spec.tsx.snap [hunk 1]
```diff
@@ -1135,6 +1135,31 @@ exports[`content in browser matches snapshot for logged out 1`] = `
...
+  height: 6rem;
+  max-width: 128rem;
+  margin: 0 auto;
+}
+
+.c2 {
+  display: block;
+  width: auto;
+  height: 3.5rem;
+}
```
Comment (jivey) [inline src/app/components/NavBar/__snapshots__/index.spec.tsx.snap:1161]: Something with the snapshots is duplicating styles, there's multiple copies of the same rules in this spec...

**Review by TomWoodward — APPROVED** [review]

**Pull request merged**